Module/uti/adventurer_fe001.uti.json

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
I think all of these new items you created are using less-descriptive resrefs than expected. Please review & modify them to be more clear of what they are. For example, this one is named "Scientist Outfit Female" but the resref indicates it's some kind of adventurer armor. I suspect it's a copy/paste situation but please address.